tests/test_packages/test_skills/test_thermometer.py

```diff
@@ -142,16 +201,31 @@ def test_themrometer(self, pytestconfig):
                 "fetchai/oef:0.1.0",
             ],
             stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
             env=os.environ.copy(),
         )
 
-        # check the gym run ends
+        tty_read_thread = threading.Thread(target=_read_tty, args=(process_one,))
+        tty_read_thread.start()
```

Comment:
these threads are never stopped?

Reply:
These threads should stop automatically. At least this is what @marcofavorito told me :)

Reply:
Is that so @marcofavorito . If so, we should have a comment. It's not obvious that they must stop automatically and the next person reviewing this might be confused.